gramex/apps/ui/__init__.py

```diff
@@ -133,7 +133,8 @@ def sass2(handler, path: str = join(ui_dir, 'gramexui.scss')):
         with open(scss_path, 'w', encoding='utf-8') as handle:
             handle.write('\n'.join(content))
         # Compile SASS file. Allow @import from template dir, UI dir, Bootstrap dir
-        proc = yield gramex.service.threadpool.submit(subprocess.run, [
+        # proc = yield gramex.service.threadpool.submit(subprocess.run, [
+        proc = subprocess.run([
```

Comment:
- Rename sass2 to _sass without the coroutine
- Make sass2 yield `gramex.service.threadpool.submit(_sass, ...)`

Reply:
Done, @sanand0 . Please take a look.